tools/subgraph-example/subgraph/package.json

```diff
@@ -11,8 +11,8 @@
     "test": "graph test"
   },
   "dependencies": {
-    "@graphprotocol/graph-cli": "0.36.1",
+    "@graphprotocol/graph-cli": "0.53.0",
```

Comment:
Q: Nice jump, any improvements to note or just keeping up?

Reply:
Just keeping up.

Reply:
👍